tests/helics/CMakeLists.txt

```diff
@@ -30,5 +30,19 @@ endif()
 add_subdirectory(apps)
 
 add_subdirectory(performance_tests)
+endif()
 
-endif()
\ No newline at end of file
+# Tests for other CMake projects including and using HELICS
+add_test(find-package-tests ${CMAKE_CTEST_COMMAND} -C ${CMAKE_BUILD_TYPE}
```

Comment:
This is going to fail on Visual studio with Cmake 3.14+ since it doesn't have the architecture, we would need to do something like what we doing the BuildZMQ to account for that